index.html

```diff
@@ -21,14 +24,186 @@
             <img class="bars" src="img/ic_menu.png" alt="">
         </nav>
     </div>
+
     <!-- toolbar section ends -->
 
+
+
     <!-- headline section starts -->
+
     <section class="header">
-        <h1 class="name">Lil-Snow</h1>
-        <p class="about">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.</p>
-        <button class="btn">Start collaboration</button>
+        <h1 class="name title_font">Lil-Snow</h1>
+        <p class="feature">Hello! I am a software developer! I can help you build a product, feature or website. Taka a
+            look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+        </p>
+        <button class="main_btn">Start collaboration</button>
     </section>
+
     <!-- headline section ends -->
+
+
+
+    <!-- work section starts -->
+    <section class="work">
+        <h1 class="title_font projectH1">Projects</h1>
```

Comment:
- [ ] Kindly use another header tag for the title. `h1` tag is mostly use to give a clear indication of the content of the page. 😄